HTML+CSS+JS/Motion Background/Readme.md

```diff
@@ -0,0 +1,8 @@
+# issue
+done with the motion background (html + css )
+# used 
+you can use any component as per your wish even from google just copy the path and paste it in place the element currently used.
+# demo
+
+https://user-images.githubusercontent.com/66902249/121811829-395b5100-cc83-11eb-93fc-8ee051550c03.mp4
```

Comment:
To show a GIF or mp4 in markdown enclose it like this-
`[Name of component](Link here)`

Then it will show like this-

[Background Motion](https://user-images.githubusercontent.com/66902249/121811829-395b5100-cc83-11eb-93fc-8ee051550c03.mp4)